clang-plugin/Indexer.cpp

```diff
@@ -1208,20 +1203,25 @@ class IndexConsumer : public ASTConsumer,
       // Just take the first token.
       CXXOperatorCallExpr* op = dyn_cast<CXXOperatorCallExpr>(e);
       loc = op->getOperatorLoc();
-      NormalizeLocation(&loc);
     } else if (MemberExpr::classof(calleeExpr)) {
       MemberExpr* member = dyn_cast<MemberExpr>(calleeExpr);
       loc = member->getMemberLoc();
-      NormalizeLocation(&loc);
     } else if (DeclRefExpr::classof(calleeExpr)) {
       // We handle this in VisitDeclRefExpr.
       return true;
     } else {
-      if (callee->getLocEnd() != startLoc) {
+      if (callee->getLocEnd() != callee->getLocStart()) {
```

Comment:
Shouldn't this be `==`? We're skipping if `end != start`, while I'd expect to skip when the start and end are the same.

Reply:
I decided to pull out this whole branch. I don't think any of it makes any sense anyway. It's testing |callee|, which is the callee declaration. It would only make sense to look at calleeExpr.